### PR TITLE
Add more pages including previous branches

### PR DESCRIPTION
**Pull request by Cerdtruir**
Add more pages including previous branches from my book list project

Preview: https://htmlpreview.github.io/?https://github.com/Cerdtruir/My-Book-List/blob/Add-more-pages/index.html

### CODE REVIEW

**Comment by Cerdtruir** [comment]
Add screenshot and links to readme

**Comment by Cerdtruir** [comment]
@ikraamg You asked me to use an array as the data structure in the previous pr. I changed it to a class object in another branch. Does it still make more sense to use an array with the class?

File: script.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
+class Books {
+  addBook(title, author) {
+    const id = title + author;
+    if (!title || !author) {
+      return;
+    }
+    this[id] = { title, author };
+  }
+
+  remveBook(id) {
```
Comment (ikraamg) [inline script.js]: I don't think you are using this method right?

File: script.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+}
+
+// eslint-disable-next-line no-unused-vars
+function addBook() {
+  const addTitle = document.getElementById('title').value;
+  const addAuthor = document.getElementById('author').value;
+  bookList.addBook(addTitle, addAuthor);
+  document.getElementById('form').reset();
+  storeBooks();
+  updateHTML();
```
Comment (ikraamg) [inline script.js:66]: I think you should redirect to the `listPage()` after adding a book.

**Review by ikraamg — APPROVED** [review]
Nice work here 👍 

Noted two things that should be changed.

**Pull request merged**